src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { Post } from '../../types/Post';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo key={post.id} post={post} />
+    ))}
+  </div>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots

Reply:
not fixed
![image](https://user-images.githubusercontent.com/98764641/234225651-f9b1c7de-92ab-4433-9bad-78da6785468d.png)